README.md

```diff
@@ -10,3 +10,30 @@ We got:
 4. Wang Affinity for spectral clustering. Speaker Diarization with LSTM, Wang et al, https://arxiv.org/abs/1710.10468, https://github.com/wq2012/SpectralCluster/blob/master/spectralcluster/spectral_clusterer.py
 5. Basic Spectral Clustering implementation
 6. Jupyter Notebook showcasing affinity for short and long English audio
+
+# CLI commands
+
+Dataset construction
+```
+python -m dataset.diarization dataset -i folder_with_wavs/ -o dataset.json --processes 5
```

Comment:
let's rename `--processes` to `--workers`/`--num-workers` as used in `convasr/train.py`?